src/accounts.rs

```diff
@@ -480,7 +478,9 @@ impl Config {
             };
             return Ok(cfg);
         }
-        let lock_task = Self::create_lock_task(dir.to_path_buf()).await?;
+        let Ok(lock_task) = Self::create_lock_task(dir.to_path_buf()).await else {
+            bail!("Delta Chat is already running. To use Delta Chat, you must first close the existing Delta Chat process, or restart your device.\n\n(accounts.lock file is already locked)");
```

Comment:
I suggest to move this to `create_lock_task()`, there are other reasons why it may fail, even if they are unlikely